### PR TITLE
fix(Remediation): apply new loading approach

### PR DESCRIPTION
**Pull request by mkholjuraev**
TICKET: https://issues.redhat.com/browse/SPM-1220.

This PR affets all remediation buttons, modals used in Patch UI. It is intended to load remediations using **AsyncComponents**.

### CODE REVIEW

**Review by bastilian — APPROVED** [review]
Works for me! I did notice some minor things

When the button or the wizard are loaded they show the spinner which looks a bit odd.

https://user-images.githubusercontent.com/7757/158174966-315e3318-2a45-4a76-9f40-1ea301af25e6.mp4

 In Compliance has a disabled "Remediate" button as the fallback, so the table/toolbar don't jump as much. Maybe for the wizard a empty span or something would make sense?

File: src/PresentationalComponents/TableView/__snapshots__/TableView.test.js.snap [hunk 1]
```diff
@@ -1,19 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`TableView Should open remediation modal 1`] = `undefined`;
```
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply src/PresentationalComponents/TableView/__snapshots__/TableView.test.js.snap:3]: Nope :). I will fix this

**Comment by mkholjuraev** [comment]
> Works for me! I did notice some minor things
> 
> When the button or the wizard are loaded they show the spinner which looks a bit odd.

The loading spinner is on purpose. Patch had before to show that system issues are being loaded from backend. We had to fetch all issues per system and it took a lot of time.

>  Kapture.2022-03-14.at.13.45.14.mp4 
> In Compliance has a disabled "Remediate" button as the fallback, so the table/toolbar don't jump as much. Maybe for the wizard a empty span or something would make sense?

Spinner did help to get rid of the jitter. Thanks @bastilian for the suggestion.

**Pull request merged**